e2e/token-vault-suites/src/utils/async-events.ts

```diff
@@ -16,6 +16,25 @@ export function asyncEvents(page) {
         page.getByRole('link', { name: text }).click(),
       ]);
     },
+    async getTokens(origin, clientId) {
```

Comment:
Since dealing with `localStorage` using Playwright is a bit complicated, I'm pulling it out into a utility for getting tokens from the web app in order to apply assertions to them in the test.